nav2_behavior_tree/plugins/action/compute_path_to_pose_action.cpp

```diff
@@ -43,6 +43,20 @@ BT::NodeStatus ComputePathToPoseAction::on_success()
   return BT::NodeStatus::SUCCESS;
 }
 
+BT::NodeStatus ComputePathToPoseAction::on_aborted()
+{
+  geometry_msgs::msg::PoseStamped empty_path;
```

Comment:
Wrong type :wink: This should be a Path not a pose stamped. 

https://app.circleci.com/pipelines/github/ros-planning/navigation2/7718/workflows/ff4e7675-61ac-482f-baad-bc0d3fde29ec/jobs/26788

CI failed on this (also @facontidavide great user protective error! That saved us a bunch of time and energy)

Reply:
you are welcome :)